### PR TITLE
Stop agent with response

### PR DESCRIPTION
**Pull request by wbooth**

### CODE REVIEW

File: gocat/util/utility.go [hunk 1]
```diff
@@ -64,3 +64,8 @@ func removeWhiteSpace(input string) string {
 		return r
 	}, input)
 }
+
+func StopProcess(pid int) {
+	proc, _ := os.FindProcess(pid)
+	_ = proc.Kill()
```
Comment (privateducky) [inline gocat/util/utility.go]: remove _ variable, since it is unused.

**Comment by privateducky** [comment]
can you recompile the binaries (ensuring you have the latest code that we merged Friday in this repo) and ensure the operation runs as anticipated?

**Comment by wbooth** [comment]
this branch already includes the changes from Friday (https://github.com/mitre/sandcat/blob/stop-agent-response/gocat/api/api.go#L33 for example)

File: gocat/util/utility.go (hunk 1 above)
Comment (wbooth) [reply gocat/util/utility.go]: it is unused but is included to explicitly ignore the error of a failed process kill in the same convention as used other places (see https://github.com/mitre/sandcat/blob/master/gocat/util/utility.go#L25)

**Pull request merged**